client/a8c-for-agencies/components/a4a-number-input-v2/index.tsx

```diff
@@ -1,4 +1,3 @@
-// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
```

Comment:
This is not the right way, fixed it by adding it here: https://github.com/Automattic/wp-calypso/pull/99394/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5